### DISCUSSION TITLE
Suggestion | Privacy Mode For Linux

### DISCUSSION THREAD

**Discussion by stuff97** [rustdesk/rustdesk#6739] [Feature Request]
Hi,

I'd like to suggest adding privacy mode for Linux Operating Systems since I noticed that it's not there and I have a suggestion which will at least work for laptops. In order to work somewhat privately I use a workaround where I use `xbacklight` command from the `acpilight` in my Archlinux laptops (Thinkpad / Dell) to completely turn off the laptop's display.

I'm not sure if it's doable but this could be used to set the screen brightness to 0 `xbacklight -set 0` and make sure it stays that way unless privacy mode is disabled again.

**Comment by lazebird** [comment]
hope to have privacy mode for linux too

**Comment by chan-yuu** [comment]
good point, marked.
